openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -581,6 +590,264 @@ def create_heketi_volume_with_name_and_wait(
 
         return h_volume_info
 
+    def cleanup_heketi_blockhosting_volumes(
+            self, bhv_initial, skip_cleanup_bv=False, raise_on_error=False):
+        """Delete heketi blockhosting volumes and blockvolumes.
+
+        Args:
+            initial_vols (list): Volumes present at initial
+            skip_cleanup_bv (bool): skip cleanup or not.
+        """
+        bhv_final = list(
+            get_block_hosting_volume_list(
+                self.heketi_client_node, self.heketi_server_url).keys())
+
+        diff_volumes = list(set(bhv_final) - set(bhv_initial))
+        for volume in diff_volumes:
+            h_vol_info = heketi_volume_info(
+                self.heketi_client_node, self.heketi_server_url,
+                volume, json=True)
+            if not skip_cleanup_bv and h_vol_info.get("block", {}):
+                for block_vol in (
+                        h_vol_info.get(
+                            "blockinfo", {}).get("blockvolume", {})):
+                    heketi_blockvolume_delete(
+                        self.heketi_client_node, self.heketi_server_url,
+                        block_vol)
+            heketi_volume_delete(
+                self.heketi_client_node, self.heketi_server_url,
+                volume, raise_on_error=raise_on_error)
+
+    def create_heketi_volumes_in_batch(
+            self, vol_count, size=1, batch_amount=8, timeout=300, interval=5,
+            block=False, skip_cleanup=False, **kwargs):
+        """Creates heketi volume in batches.
+
+        Args:
+            heketi_client_node (str): Node on which cmd has to executed.
+            heketi_server_url (str): Heketi server url.
+            vol_count (int): Count of volumes to create.
+            size (str): Volume size.
+            batch_amount (int): Amount of volumes to be created in one batch.
+            timeout (int): Timeout for entire volume count to create.
+            interval (int): Time interval between each interation.
+            skip_cleanup (bool): skip cleanup or not.
+
+        Returns:
+            list: list of heketi volume id's.
+
+        Kwargs:
+            The keys, values in kwargs are:
+                - block : (bool)
```

Comment:
As we are passing block as separate value as function parameter, move this to `Args` section

Reply:
Done